test/src/test/java/hudson/slaves/AgentInboundUrlTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2021, CloudBees, Inc.
```

Comment:
Really?

Reply:
I'm going to be honest, I don't know what to put here. I pulled this from a recently added file and bumped the date. I'm not used to writing licenses

Reply:
You wrote it, you own it. Put your own name there ;)

Reply:
I believe I figured it out 😅